include/dlpack/dlpack.h

```diff
@@ -216,13 +243,66 @@ typedef struct DLManagedTensor {
    *   which DLManagedTensor is used in the framework. It can also be NULL.
    */
   void * manager_ctx;
-  /*! \brief Destructor signature void (*)(void*) - this should be called
-   *   to destruct manager_ctx which holds the DLManagedTensor. It can be NULL
-   *   if there is no way for the caller to provide a reasonable destructor.
-   *   The destructors deletes the argument self as well.
+  /*!
+   * \brief Destructor - this should be called
```

Comment:
nit picking: formatting is a little messed up (double spaces, rows of different length)

Reply:
great point, fixed for this particular case. we can do further cleanups